src/playbacks/html5_video/html5_video.js

```diff
@@ -140,12 +140,19 @@ export default class HTML5Video extends Playback {
     this._handleBufferingEvents()
     this.trigger(Events.PLAYBACK_LOADEDMETADATA, {duration: e.target.duration, data: e})
     this._updateSettings()
+    this._checkHighDefinition(e)
     const autoSeekFromUrl = typeof(this._options.autoSeekFromUrl) === 'undefined' || this._options.autoSeekFromUrl
     if (this.getPlaybackType() !== Playback.LIVE && autoSeekFromUrl) {
       this._checkInitialSeek()
     }
   }
 
+  _checkHighDefinition(e) {
+    const currentHeight = e.target.videoHeight
+    this.highDefinition = currentHeight >= 720
```

Comment:
That would be a good idea to have a base _checkHighDefinition method that other playbacks would override.